Topic 3/src/test/java/bootcamp/exercises/ConvertToTextTest.java

```diff
@@ -0,0 +1,33 @@
+package bootcamp.exercises;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class ConvertToTextTest {
+
+	ConvertToText number = new ConvertToText();
+
+	@Test
+	public void testConvertToText() {
+		assertEquals(number.convert("73,30"), "seventy-three and 30/100 dollars.");
+		assertEquals(number.convert("102,47"), "one hundred two and 47/100 dollars.");
+		assertEquals(number.convert("2523.04"), "two thousand five hundred twenty-three and 04/100 dollars.");
+		assertEquals(number.convert("12523.04"), "twelve thousand five hundred twenty-three and 04/100 dollars.");
+		assertEquals(number.convert("20020.04"), "twenty thousand twenty and 04/100 dollars.");
+		assertEquals(number.convert("2000.04"), "two thousand  and 04/100 dollars.");
+		assertEquals(number.convert("130540.04"), "one hundred and thirty thousand five hundred forty and 04/100 dollars.");
+		assertEquals(number.convert("102130540.04"),
+				"one hundred two million one hundred and thirty thousand five hundred forty and 04/100 dollars.");
+	}
+
+	/**
+	 * If the entered number is out of range, shoot me an exception.
+	 */
+	@Test(expected = OverflowException.class)
+	public void testOverflow() {
+		number.convert("9999999999");
+		number.convert("-1");
+	}
+
```

Comment:
[High] There are missing scenarios to tests. For example:
1. What happens if the user write invalid number: "45435sd,j56" or "twenty" or "" or <code>null</code> or "234.234" or "3.0394,9" or "3445,3345,70"
2. What happens if the user write integer numbers: "4577"

Remember that here you want to cover all scenarios that make sure your code is working as expected.